From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 15 Feb 2026 12:06:33 +0100
Subject: Revert "portal: Use Rust based GUI variants by default"

We need newer ashpd in Debian for that

This reverts commit 0b9b65a17ce063a29fd8f4cfb7f20367d2f6d5f8.
---
 data/phosh-portals.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/phosh-portals.conf b/data/phosh-portals.conf
index 3bf377d..600c139 100644
--- a/data/phosh-portals.conf
+++ b/data/phosh-portals.conf
@@ -1,5 +1,5 @@
 [preferred]
-default=phrosh;phosh;gtk;
+default=phosh;phrosh;gtk;
 org.freedesktop.impl.portal.Access=phosh-shell;
 org.freedesktop.impl.portal.Background=none;
 org.freedesktop.impl.portal.Clipboard=none;
